website/docs/Installation/windows-install/installing-podman-with-openshift-local.md

```diff
@@ -21,6 +21,7 @@ Consider installing the Podman Preset in OpenShift Local rather than [installing
 #### Prerequisites
 
 * The user is a member of the *Hyper-V Administrators* group. The user can add a virtual machine, start or stop this virtual machine, and access the resources related to this virtual machine.
+* Windows 10 or 11, Enterprise or Professional edition
```

Comment:
should we mention the exact version for windows 10? in the CRC docs its mentioned as `Windows 10 Fall Creators Update (version 1709) or later`

ref: https://crc.dev/crc/#_microsoft_windows